### PR TITLE
Update #2

### PR DESCRIPTION
**Pull request by noolbar**
I updated the codes for PureScript 0.12.x.
Most of them were fixed, but I could not do `halogen-server` , because `Halogen.HTML.Renderer.String` is out of date.

### CODE REVIEW

File: hello/src/Main.purs [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-import Control.Monad.Eff.Console (CONSOLE, log)
-import Prelude (Unit)
+import Prelude
+import Effect (Effect)
+import Effect.Console (logShow)
 
-main :: Eff (console :: CONSOLE) Unit
-main = log "Hello World!"
+main :: Effect Unit
+main = logShow "Hello World!"
```
Comment (bouzuya) [inline hello/src/Main.purs:8]: `logShow` -> `log` ?

File: st/src/Main.purs [hunk 2]
```diff
@@ -1,38 +1,34 @@
...
-  ref <- newSTRef 1
-  modifySTRef ref \x -> x * 2
-  modifySTRef ref \x -> x * 2
-  modifySTRef ref \x -> x * 2
-  readSTRef ref
+example1 :: Effect Unit
+example1 = do
+  ref <- Ref.new 123
+  n1 <- Ref.read ref
+  logShow $ show n1
```
Comment (bouzuya) [inline st/src/Main.purs]: `logShow $ show n1` -> (`logShow n1` or `log $ show n1`)

File: hello/src/Main.purs (hunk 1 above)
Comment (noolbar) [reply hello/src/Main.purs:8]: Thank you for reviewing. Should I need to rewrite it?

Comment (bouzuya) [reply hello/src/Main.purs:8]: I will fix it.

**Comment by bouzuya** [comment]
Thanks

**Pull request merged**